src/interfaces/EditInterface/EditInterface.js

```diff
@@ -20,7 +20,10 @@ import {
   TextField,
 } from '@folio/stripes/components';
 
-import { isURLValid } from '../../Utils/Validate';
+import {
+  isURLValid,
+  Required,
```

Comment:
`Required` - is a quite obsolete code, I understand that it's hard to know it for you, but in any case - could you please use `validateRequired` from `@folio/stripes-acq-components` ? Thanks

Reply:
Yep, thanks for the comment
I correctly understand that i must to make new commit with this changes?